agent-crack/doc/crack.html

```diff
@@ -1,71 +0,0 @@
-<HTML>
```

Comment:
What is reason to remove this document? Even if we do not distribute it, I think it is useful as source of information about SCR agent on top of libcrack